server/core/src/irods_report_plugins_in_json.cpp

```diff
@@ -26,11 +32,22 @@ namespace irods {
              itr != plugin_list.end();
              ++itr)
         {
+
+            const auto filepath = fmt::format("{}lib{}.so", plugin_home, *itr);
+
+            char checksum[NAME_LEN]{};
+            int ret = chksumLocFile(filepath.c_str(), checksum, irods::SHA256_NAME.c_str());
+
+            if (ret < 0) {
+                log_api::error("ChecksumLocFile Failure - Failed to calculate checksum for plugin: [{}]", *itr);
```

Comment:
Let's include the error code in the log message. If this branch is ever taken, it'd be good to know what happened.

Reply:
So something like this?

```c++
log_api::error("ChecksumLocFile Failure {} - Failed to calculate checksum for plugin: [{}]", ret, *itr);
```